doc/spec-files.rst

```diff
@@ -626,22 +627,26 @@ changing them to give each Analysis object a unique name::
 
 Now call the MERGE method to process the three Analysis objects::
 
-    MERGE( (foo_a, 'foo', 'foo'), (bar_a, 'bar', 'bar'), (zap_a, 'zap', 'zap') )
+    MERGE( (foo_a, 'foo', os.path.join('foo', 'foo'),
+           (bar_a, 'bar', os.path.join('bar', 'bar'),
+           (zap_a, 'zap', os.path.join('zap', 'zap') )
```

Comment:
As per the comment above, both the original and the new example are correct (and coversely, incorrect at the same time), depending on the context; i.e., are executables `onedir` or `onefile` ones. 

That said, `MERGE` doesn't really make sense with `onedir` executables, even if it is technically possible and even though we even have tests for various mixed-mode scenarios:
- https://github.com/pyinstaller/pyinstaller/blob/develop/tests/functional/specs/test_multipackage1.spec
- https://github.com/pyinstaller/pyinstaller/blob/develop/tests/functional/specs/test_multipackage2.spec
- https://github.com/pyinstaller/pyinstaller/blob/develop/tests/functional/specs/test_multipackage3.spec
- https://github.com/pyinstaller/pyinstaller/blob/develop/tests/functional/specs/test_multipackage4.spec
- https://github.com/pyinstaller/pyinstaller/blob/develop/tests/functional/specs/test_multipackage5.spec

But for `onedir` executables, `MERGE` brings no benefits and forces the executable into onefile semantics (copying the files to temporary directory on each run). For `onedir` executables, [a shared COLLECT](https://www.zacoding.com/en/post/pyinstaller-create-multiple-executables/) should be used instead.